packages/recommend-vdom/src/FacetsView.tsx

```diff
@@ -0,0 +1,33 @@
+/** @jsx createElement */
+import { RecordWithFacets } from '@algolia/recommend-core/src';
+
+import {
+  RecommendClassNames,
+  RecommendTranslations,
+  Renderer,
+  FacetsViewProps,
+} from './types';
+import { cx } from './utils';
+
+export function createFacetsView({ createElement }: Renderer) {
+  return function FacetsView<TItem extends RecordWithFacets>(
+    props: FacetsViewProps<TItem, RecommendTranslations, RecommendClassNames>
+  ) {
+    return (
+      <div
+        className={cx('auc-Recommend-container', props.classNames.container)}
+      >
+        <ol className={cx('auc-Recommend-list', props.classNames.list)}>
+          {props.items.map((item) => (
+            <li
+              key={item.facetValue}
```

Comment:
Should we also use the facetName in the key, so that if we render multiple `TrendingFacets` side by side (e.g. one for `mainColor` one for `secondaryColor`) we don't have two FacetsViews with the same `key`? 

```suggestion
              key={item.facetKey + '_' + item.facetValue}
```

Reply:
React keys are scoped to their list (`map()`) so the facet key shouldn't be needed.